substratevm/mx.substratevm/suite.py

```diff
@@ -1078,7 +1091,7 @@
                     "com.oracle.svm.core.graal.thread             to jdk.internal.vm.compiler",
                     "com.oracle.svm.core.classinitialization      to jdk.internal.vm.compiler",
                     "com.oracle.svm.truffle.api                   to org.graalvm.truffle",
-                    "* to org.graalvm.nativeimage.driver,org.graalvm.nativeimage.configure,org.graalvm.nativeimage.librarysupport,org.graalvm.nativeimage.llvm,org.graalvm.nativeimage.agent.jvmtibase,org.graalvm.nativeimage.agent.tracing,org.graalvm.nativeimage.agent.diagnostics,com.oracle.svm.svm_enterprise",
+                    "* to  org.graalvm.nativeimage.pointsto,org.graalvm.nativeimage.driver,org.graalvm.nativeimage.configure,org.graalvm.nativeimage.librarysupport,org.graalvm.nativeimage.llvm,org.graalvm.nativeimage.agent.jvmtibase,org.graalvm.nativeimage.agent.tracing,org.graalvm.nativeimage.agent.diagnostics,com.oracle.svm.svm_enterprise",
```

Comment:
Why is this export necessary? Seems strange that `org.graalvm.nativeimage.builder` needs to export to `org.graalvm.nativeimage.pointsto`.

Reply:
Without this exporting, there is an exception thrown out at libnative-image-diagnostics-agent.so.image build time:
```
Building libnative-image-agent.so.image... [dependency SVM_CONFIGURE updated]
Building libnative-image-diagnostics-agent.so.image... [dependency JVMTI_AGENT_BASE updated]
Exception in thread "main" com.oracle.svm.core.util.VMError$HostedError: org.graalvm.nativeimage.base.option.OptionParsingException: java.lang.IllegalAccessException: class org.graalvm.nativeimage.base.option.CommonOptionParser (in module org.graalvm.nativeimage.pointsto) cannot access class com.oracle.svm.hosted.SVMHost_OptionDescriptors (in module org.graalvm.nativeimage.builder) because module org.graalvm.nativeimage.builder does not export com.oracle.svm.hosted to module org.graalvm.nativeimage.pointsto
        at org.graalvm.nativeimage.builder/com.oracle.svm.core.util.VMError.shouldNotReachHere(VMError.java:72)
        at org.graalvm.nativeimage.builder/com.oracle.svm.core.option.SubstrateOptionsParser.collectOptions(SubstrateOptionsParser.java:125)
        at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.option.HostedOptionParser.collectOptions(HostedOptionParser.java:63)
        at org.graalvm.nativeimage.driver/com.oracle.svm.driver.APIOptionHandler.extractOptions(APIOptionHandler.java:122)
        at org.graalvm.nativeimage.driver/com.oracle.svm.driver.APIOptionHandler.<init>(APIOptionHandler.java:115)
        at org.graalvm.nativeimage.driver/com.oracle.svm.driver.NativeImage.<init>(NativeImage.java:820)
        at org.graalvm.nativeimage.driver/com.oracle.svm.driver.NativeImage.lambda$static$19(NativeImage.java:1457)
        at org.graalvm.nativeimage.driver/com.oracle.svm.driver.NativeImage.build(NativeImage.java:1490)
        at org.graalvm.nativeimage.driver/com.oracle.svm.driver.NativeImage.performBuild(NativeImage.java:1473)
        at org.graalvm.nativeimage.driver/com.oracle.svm.driver.NativeImage.main(NativeImage.java:1460)
Caused by: org.graalvm.nativeimage.base.option.OptionParsingException: java.lang.IllegalAccessException: class org.graalvm.nativeimage.base.option.CommonOptionParser (in module org.graalvm.nativeimage.pointsto) cannot access class com.oracle.svm.hosted.SVMHost_OptionDescriptors (in module org.graalvm.nativeimage.builder) because module org.graalvm.nativeimage.builder does not export com.oracle.svm.hosted to module org.graalvm.nativeimage.pointsto
        at org.graalvm.nativeimage.pointsto/org.graalvm.nativeimage.base.option.CommonOptionParser.collectOptions(CommonOptionParser.java:169)
        at org.graalvm.nativeimage.builder/com.oracle.svm.core.option.SubstrateOptionsParser.collectOptions(SubstrateOptionsParser.java:123)
        ... 8 more
Caused by: java.lang.IllegalAccessException: class org.graalvm.nativeimage.base.option.CommonOptionParser (in module org.graalvm.nativeimage.pointsto) cannot access class com.oracle.svm.hosted.SVMHost_OptionDescriptors (in module org.graalvm.nativeimage.builder) because module org.graalvm.nativeimage.builder does not export com.oracle.svm.hosted to module org.graalvm.nativeimage.pointsto
        at java.base/jdk.internal.reflect.Reflection.newIllegalAccessException(Reflection.java:361)
        at java.base/java.lang.reflect.AccessibleObject.checkAccess(AccessibleObject.java:591)
        at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:481)
        at org.graalvm.nativeimage.pointsto/org.graalvm.nativeimage.base.option.CommonOptionParser.collectOptions(CommonOptionParser.java:167)
        ... 9 more
Exception in thread "main" com.oracle.svm.core.util.VMError$HostedError: org.graalvm.nativeimage.base.option.OptionParsingException: java.lang.IllegalAccessException: class org.graalvm.nativeimage.base.option.CommonOptionParser (in module org.graalvm.nativeimage.pointsto) cannot access class com.oracle.svm.hosted.SVMHost_OptionDescriptors (in module org.graalvm.nativeimage.builder) because module org.graalvm.nativeimage.builder does not export com.oracle.svm.hosted to module org.graalvm.nativeimage.pointsto
        at org.graalvm.nativeimage.builder/com.oracle.svm.core.util.VMError.shouldNotReachHere(VMError.java:72)
        at org.graalvm.nativeimage.builder/com.oracle.svm.core.option.SubstrateOptionsParser.collectOptions(SubstrateOptionsParser.java:125)
        at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.option.HostedOptionParser.collectOptions(HostedOptionParser.java:63)
        at org.graalvm.nativeimage.driver/com.oracle.svm.driver.APIOptionHandler.extractOptions(APIOptionHandler.java:122)
        at org.graalvm.nativeimage.driver/com.oracle.svm.driver.APIOptionHandler.<init>(APIOptionHandler.java:115)
        at org.graalvm.nativeimage.driver/com.oracle.svm.driver.NativeImage.<init>(NativeImage.java:820)
        at org.graalvm.nativeimage.driver/com.oracle.svm.driver.NativeImage.lambda$static$19(NativeImage.java:1457)
        at org.graalvm.nativeimage.driver/com.oracle.svm.driver.NativeImage.build(NativeImage.java:1490)
        at org.graalvm.nativeimage.driver/com.oracle.svm.driver.NativeImage.performBuild(NativeImage.java:1473)
        at org.graalvm.nativeimage.driver/com.oracle.svm.driver.NativeImage.main(NativeImage.java:1460)
Caused by: org.graalvm.nativeimage.base.option.OptionParsingException: java.lang.IllegalAccessException: class org.graalvm.nativeimage.base.option.CommonOptionParser (in module org.graalvm.nativeimage.pointsto) cannot access class com.oracle.svm.hosted.SVMHost_OptionDescriptors (in module org.graalvm.nativeimage.builder) because module org.graalvm.nativeimage.builder does not export com.oracle.svm.hosted to module org.graalvm.nativeimage.pointsto
        at org.graalvm.nativeimage.pointsto/org.graalvm.nativeimage.base.option.CommonOptionParser.collectOptions(CommonOptionParser.java:169)
        at org.graalvm.nativeimage.builder/com.oracle.svm.core.option.SubstrateOptionsParser.collectOptions(SubstrateOptionsParser.java:123)
        ... 8 more
Caused by: java.lang.IllegalAccessException: class org.graalvm.nativeimage.base.option.CommonOptionParser (in module org.graalvm.nativeimage.pointsto) cannot access class com.oracle.svm.hosted.SVMHost_OptionDescriptors (in module org.graalvm.nativeimage.builder) because module org.graalvm.nativeimage.builder does not export com.oracle.svm.hosted to module org.graalvm.nativeimage.pointsto
        at java.base/jdk.internal.reflect.Reflection.newIllegalAccessException(Reflection.java:361)
        at java.base/java.lang.reflect.AccessibleObject.checkAccess(AccessibleObject.java:591)
        at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:481)
        at org.graalvm.nativeimage.pointsto/org.graalvm.nativeimage.base.option.CommonOptionParser.collectOptions(CommonOptionParser.java:167)
        ... 9 more

Building libnative-image-diagnostics-agent.so.image: Failed due to error: 1

Building libnative-image-agent.so.image: Failed due to error: 1
Building libnative-image-agent.so.image failed
Building libnative-image-diagnostics-agent.so.image failed
2 build tasks failed

```